saleor/graphql/account/mutations/account.py

```diff
@@ -96,11 +103,19 @@ def clean_input(cls, info, instance, data, input_cls=None):
                 }
             )
 
+        data["channel"] = clean_channel(
+            data.get("channel"),
+            error_channel_not_defined=AccountErrorCode.MISSING_CHANNEL_SLUG.value,
+            error_for_channel_doesnt_exist=AccountErrorCode.NOT_FOUND.value,
+            error_for_channel_inactive=AccountErrorCode.CHANNEL_INACTIVE.value,
```

Comment:
We could try to only pass the class name as a signle argument e.g. `error_class=AccountError` - the error codes should be the same in each case.